CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+# [1.5.0] - 2023-03-28
+
+### Changed
+
+- Common: Change signature of `set_link_suggestion` function to take an attributes argument
```

Comment:
Is this a breaking change for users of the library (requiring a major version update)?